### ISSUE TITLE
Add lowercase version of all the functions

### ISSUE THREAD

**Issue by alepuccetti** [CartoDB/carto-spatial-extension#127]
BigQuery supports all its native functions to be upper or lower case. It would be nice to have the same support for here too.

Is this something that could be valued by others too?

This probably is valid for snowflake too but I don't use it so I am not sure.

**Comment by francois-baptiste** [comment]
I agree that would be nice. However, from my perspective the change needs to be taken globally at the bigquery level.
That way mixed upper or lower case would be also supported without code duplication.
